### PR TITLE
Remove duplicate picture versions

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: app/jobs/europeana/picture_versions_cleanup_job.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Europeana
+  class PictureVersionsCleanupJob < ActiveJob::Base
```
Comment (rwd) [inline app/jobs/europeana/picture_versions_cleanup_job.rb]: Could we add a note here as a reminder that this is only needed in the code base temporarily, until our live deployments have all had the job executed, after which time this will no longer be needed.

File: deploy/bluemix/config/database.yml [hunk 2]
```diff
@@ -9,3 +9,4 @@ postgres_url = service_config['credentials']['uri']
 %>
 production:
   url: "<%= postgres_url %>"
+  pool: 20 # Puma defaults to 16 threads per instance. +4 for leeway.
```
Comment (rwd) [inline deploy/bluemix/config/database.yml]: Is this here intentionally? Seems unrelated to the rest of the work here re picture versions.

Comment (lbiedinger) [reply deploy/bluemix/config/database.yml]: Not related to the PR, but came in as part of the hotfixes I mentioned in Jira. I believe it was helpful with the database connection issue, but it didn't solve the problem entirely.
